modules/aws_eks/aws_eks.py

```diff
@@ -35,8 +39,7 @@ def post_delete(self, module_idx: int) -> None:
         purge_opta_kube_config(layer=self.layer)
 
     def pre_hook(self, module_idx: int) -> None:
-        cluster_name = get_cluster_name(self.layer.root())
-        if cluster_name is not None:
+        if does_cluster_exist(self.layer.root()):
```

Comment:
what if the cluster is gone but the cloudwatch log group was not deleted, are we still gonna delete them prior creating a new cluster?

Reply:
er... yes-- this is what these lines of code are doing, right?

Reply:
got it